db/version_set_test.cc

```diff
@@ -1232,10 +1230,8 @@ TEST_F(VersionSetTest, NonWalEditsAppliedToVersion) {
     edits.back()->AddWal(i, WalMetadata(i));
   }
   // Delete the first half of the WALs.
-  for (uint64_t i = 1; i <= kNumWals; i++) {
-    edits.emplace_back(new VersionEdit);
-    edits.back()->DeleteWal(i);
-  }
+  edits.emplace_back(new VersionEdit);
```

Comment:
And this one.

Reply:
also `autovector<std::unique_ptr<VersionEdit>> edits;`